src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -2717,6 +2718,42 @@ protected override void EndProcessing()
             Context.EngineSessionState.CurrentScope.StrictModeVersion = _version;
         }
     }
+
+    /// <summary>
+    /// Provides argument completion for StrictMode Version parameter.
+    /// </summary>
+    public class StrictModeVersionArgumentCompleter : IArgumentCompleter
+    {
+        private static readonly string[] s_StrictModeVersions = new string[] { "Latest", "3.0", "2.0", "1.0" };
```

Comment:
nitpick: static field should use `s_<camel case>`
```suggestion
        private static readonly string[] s_strictModeVersions = new string[] { "Latest", "3.0", "2.0", "1.0" };
```